src/clarity-angular/utils/focus-trap/focus-trap.directive.ts

```diff
@@ -19,7 +22,16 @@ export class FocusTrapDirective implements AfterViewInit {
     }
 
     ngAfterViewInit() {
+        this._previousActiveElement = <HTMLElement>document.activeElement;
```

Comment:
You should probably cast it to `Document`, it's an [actual interface](https://developer.mozilla.org/en-US/docs/Web/API/Document).